File: Mushka.WebApi/ClientModels/Supply/SupplyListModel.cs
using System;

namespace Mushka.WebApi.ClientModels.Supply
{
    public class SupplyListModel
    {
        public Guid Id { get; set; }
        
        public DateTime ReceivedDate { get; set; }

        public string Description { get; set; }

        public string SupplierName { get; set; }

        public int ProductsAmount { get; set; }

        public decimal Cost { get; set; }

        public decimal TotalCost { get; set; }
    }
}

File: frontend2/src/app/shared/models/supply-list.model.ts
export class SupplyList {
  id: string;
  supplierName: string;
  description: string;
  receivedDate: string;
  cost: number;
  totalCost: number;
  productsAmount: number;

  constructor(data: any) {
    Object.assign(this, data);
  }
}
